client/workers/image-loader.worker.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-restricted-globals */
+
+self.addEventListener('message', (event) => {
+  const imageURL = event.data
```

Comment:
I'd prefer if we either rename this to imgURLs (plural) or just refereed to event.data directly instead of assigning it to a variable first, since it's only used once.